woocommerce/woocommerce
# [Importer/Exporter] Fixed the value display of "Published" for children of draft variable products

### All Submissions:






### Changes proposed in this Pull Request:



By default variations only uses "Published" and "Private" statuses, so while exporting a draft variable product, all variations will shows as "Published" instead of a draft as expected.

![](https://d.pr/i/jhUvfv+)
Link to screenshot: https://d.pr/i/jhUvfv

This PR fixes this display, making easy for any user to understand that all children are also part of the draft product.

Closes #26860.

### How to test the changes in this Pull Request:

1. Create a variable product with some variations, and save your product as a draft.
2. Now export all your products, and note that your new variable product shows `-1` in the "Published" column, while all variations shows as `1`.
3. Apply this patch and export your products again, now note that shows `-1` for the variable product and its variations.
4. Try import the exported file from the previous step.
5. Check if the variations are still published.

### Other information:




### Changelog entry

> Fix - [Importer/Exporter] Fixed the value display of "Published" for children of draft variable products.
